Files/Views/LayoutModes/GenericFileBrowser.xaml

```diff
@@ -594,6 +594,8 @@
 
         <controls:DataGrid
             x:Name="AllView"
+            Background="Transparent"
+            VerticalAlignment="Stretch"
```

Comment:
Will this prevent right clicking to get the current directories right click context menu?

Reply:
With this both item context menu and "empty space" context menu will still work